dev-python/click-plugins/click-plugins-1.1.1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Module for click to enable registering CLI commands via setuptools entry-points"
+HOMEPAGE="https://github.com/click-contrib/click-plugins"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64 ~arm64 ~x86"
+SLOT="0"
+IUSE="test"
```

Comment:
remove this.

rest looks good.

Reply:
When removed:

```
Error(s) in metadata for 'dev-python/click-plugins-1.1.1':
  DEPEND: USE flag 'test' referenced in conditional 'test?' is not in IUSE
````

Reply:
because you didn't add `distutils_enable_tests pytest`

Reply:
https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html

Reply:
forgot to mention it last time :crying_cat_face:

Reply:
Thanks, didn't know about that helper, applied.

Reply:
Oh, I see that you removed the definition of the tests.  
Its a good idea to keep the custom test definition that you supplied, unless you are sure that the default test definition works for you.  
The default is a very general one, and is almost always overriden, because a lot of the tests can potentially fail because they are too strict : https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/distutils-r1.eclass#n413

Reply:
Yes, the default $(pytest -vv) works fine.

Reply:
sounds good.